Alignment/OfflineValidation/test/unit_test.yaml

```diff
@@ -148,3 +161,74 @@ validations:
                     drawHistograms:
                         drawDz: true
                         drawDxy: true
+lines:
```

Comment:
This new block `lines` is a bit unclear as it is, and we will probably add also options for the plots at the merge steps in the future, so we could add a `style` section.

Suggestion (see also comments above on lumi file and CMS label):
```
alignments:
   (...)
validations:
   (...)
style:
   CMSlabel: Internal
   trends:
        lumi: /path/to/lumi.txt
        lines:
            (...)
```